images-list

```diff
@@ -880,6 +883,13 @@ quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.11.5
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.12.0
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.12.1
 quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.12.3
+quay.io/cilium/operator-generic rancher/mirrored-cilium-operator-generic v1.12.4
+quay.io/cilium/certgen rancher/mirrored-cilium-certgen v0.1.8
+quay.io/cilium/hubble-relay rancher/mirrored-cilium-hubble-relay v1.12.4
+quay.io/cilium/hubble-ui-backend rancher/mirrored-cilium-hubble-ui-backend v0.9.2
+quay.io/cilium/hubble-ui rancher/mirrored-cilium-hubble-ui v0.9.2
+quay.io/cilium/cilium-etcd-operator rancher/mirrored-cilium-cilium-etcd-operator v2.0.7
+quay.io/cilium/clustermesh-apiserver rancher/mirrored-cilium-clustermesh-apiserver v1.12.4
```

Comment:
Lines in this file need to be sorted. This is not the correct place for these entries.

Reply:
Fixed